.perlcriticrc

```diff
@@ -14,3 +14,8 @@ severity = 5
 
 [Perl::Critic::Policy::ConsistentQuoteLikeWords]
 severity = 5
+
+[ControlStructures::ProhibitDeepNests]
+severity = 4
+add_themes = freenode
```

Comment:
Good catch here, according to documentation, add_themes makes more sense than set_themes even we could see same output.